mlrun/runtimes/mpijob/abstract.py

```diff
@@ -12,27 +12,84 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import abc
+import os
 import time
 import typing
-import os
 
+from alembic import runtime
 from kubernetes import client
-
 from mlrun.config import config
 from mlrun.execution import MLClientCtx
 from mlrun.model import RunObject
 from mlrun.runtimes.kubejob import KubejobRuntime
 from mlrun.runtimes.utils import AsyncLogWriter, RunError
-from mlrun.utils import logger, get_in
+from mlrun.utils import get_in, logger
+from mlrun.runtimes.pod import KubeResourceSpec
+
+
+class MPIResourceSpec(KubeResourceSpec):
+    def __init__(
+        self,
+        command=None,
+        py_args=None,
+        mpi_args=None,
+        image=None,
+        mode=None,
+        volumes=None,
+        volume_mounts=None,
+        env=None,
+        resources=None,
+        default_handler=None,
+        entry_points=None,
+        description=None,
+        workdir=None,
+        replicas=None,
+        image_pull_policy=None,
+        service_account=None,
+        build=None,
+        image_pull_secret=None,
+    ):
+        super().__init__(
+            command=command,
+            image=image,
+            mode=mode,
+            build=build,
+            entry_points=entry_points,
+            description=description,
+            workdir=workdir,
+            default_handler=default_handler,
+            volumes=volumes,
+            volume_mounts=volume_mounts,
+            env=env,
+            resources=resources,
+            replicas=replicas,
+            image_pull_policy=image_pull_policy,
+            service_account=service_account,
+            image_pull_secret=image_pull_secret,
+        )
+
+        self.py_args = py_args or []
```

Comment:
Take out `py_args` and keep the "regular" args interface. 
In the doc - make sure we specify that `args` are `command args`